src/main/controllers/helpers/DocumentHelpers.ts

```diff
@@ -58,21 +56,14 @@ export const combineDocuments = (...arrays: DocumentDetail[][]): DocumentDetail[
   [].concat(...arrays.filter(Array.isArray)).filter(doc => doc !== undefined);
 
 export const createDownloadLink = (file: Document): string => {
-  const mimeType = getFileExtension(file?.document_filename);
-  let downloadLink = '';
-  if (file?.document_size && file.document_mime_type && file.document_filename) {
-    const href = '/getSupportingMaterial/' + getDocId(file.document_url);
-    downloadLink =
-      `<a href='${href}' target='_blank' class='govuk-link'>` +
-      file.document_filename +
-      '(' +
-      mimeType +
-      ', ' +
-      formatBytes(file.document_size) +
-      ')' +
-      '</a>';
+  if (!file || !file.document_size || !file.document_mime_type || !file.document_filename) {
```

Comment:
Can simplify to (!file?.document_size || !file.document_mime_type || !file.document_filename)